portfolio/src/main/webapp/index.html

```diff
@@ -63,8 +63,15 @@ <h1>Omar Montiel's Portfolio</h1>
       <!-- Contact Section -->
       <div id="contact" class="section">
         <a href="resume.pdf" target="_blank">Resume</a>
-        <button onclick="getMessage()">Try Fetch</button> <!-- Fetch message from Data Servlet -->
-        <div id="message-container"></div> <!-- Display message from Data Servlet -->
+        <!-- Comment Input Form -->
+        <form action="/data" method="POST">
+          <p>Leave a comment:</p>
+          <textarea name="text-input">Say hi!</textarea>
+          <br/><br/>
```

Comment:
Consider using css to manage spacing between the different parts of the form. I like to avoid using <br> as a spacing technique because it can cause a lot of future headache/seems more hardcoded.

Reply:
Done!